app/src/main/java/com/jerboa/ui/components/comment/CommentNode.kt

```diff
@@ -233,7 +233,7 @@ fun LazyListScope.commentNodeItem(
         .commentView.counts.child_count > 0 && !isFlat
 
     increaseLazyListIndexTracker()
-    item(key = commentId) {
+    item(key = commentId, contentType = { "contentNodeItemL${node.depth}" }) {
```

Comment:
Eh it's just this one that worries me, I ll leave this one out for now and when I get some more time I ll try to get a microbench for this, bc the macrobench currently is not showing any significant perf changes. Same was for the scrolling with baselines/without but from personal expercience the baselines make a bigger impact than they are showing from the tests. So they aren't very reliable imo.

Reply:
Gotcha. Yep I'm guessing these aren't going to really matter until we have combined lists, like putting all the notification types (messages, comment replies, and mentions), in a single LazyList.